src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -110,6 +110,17 @@ class FunSpec private constructor(
     }
   }
 
+  private fun asExpressionBody(codeBlock: CodeBlock): CodeBlock? {
```

Comment:
nit: maybe make it an extension function?

Reply:
2f3bcf3c6469a3b1e4ed5f232b0f003db7c308a2